src/Service/ThemeResolver.php

```diff
@@ -41,7 +41,7 @@ class ThemeResolver
      * @param Subsite $site
      * @return array
      */
-    public function getThemeList(Subsite $site)
+    public function getThemeList(Subsite $site): array
```

Comment:
This resolves 
```text
1) SilverStripe\Subsites\Tests\SiteTreeSubsitesTest::testThemeResolverIsUsedForSettingThemeList
TypeError: SilverStripe\View\SSViewer::set_themes(): Argument #1 ($themes) must be of type array, null given, called in /var/www/html/vendor/silverstripe/subsites/src/Extensions/SiteTreeSubsites.php on line 405
```

The param in `SSViewer::set_themes()` is typed to require an array specifically, so this should return an array specifically.